src/wizard/logic-app/GenerateBuildDefinitionStep.ts

```diff
@@ -46,10 +46,10 @@ export class GenerateBuildDefinitionStep extends AzureWizardExecuteStep<IBuildDe
             const templateResource = generateTemplateResource(workflow);
             templateResources.push(templateResource);
 
-            const templateParameterDefinition = generateTemplateParameterDefinition(name);
+            const templateParameterDefinition = generateTemplate(workflow).parameters;
```

Comment:
Rather than requiring the caller to access the `parameters` property from the return value of `generateTemplate`, can you make a `generateTemplateParameters` function which does this?

Reply:
I was hoping now having to do it... :) Updated, have to change things quite a bit, let me know what you think.